tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -61,6 +61,7 @@
 	defined(CONFIG_BOARD_UBX_EVKNINAB1_NRF52832) || \
 	defined(CONFIG_BOARD_UBX_EVKNINAB3_NRF52840) || \
 	defined(CONFIG_BOARD_UBX_EVKNINAB4_NRF52833) || \
+	defined(CONFIG_BOARD_UBX_EVKANNAB4_NRF52833) || \
```

Comment:
please add this change in a separate commit

Reply:
Previously we have added a number of boards (the ones above EVKANNAB4_NRF52833) where we made this change together with adding the board folder, but I guess it is nicer to have a separate commit. Will change this.

Reply:
Done.